decidim-core/spec/validators/etiquette_validator_spec.rb

```diff
@@ -81,4 +81,19 @@ def self.model_name
       it { is_expected.to be_valid }
     end
   end
+
+  context "when the text is written in HTML" do
+    let(:body) do
+      data = File.read(Decidim::Dev.asset("avatar.jpg"))
+      encoded = Base64.encode64(data)
+
+      <<~HTML
+        <p>Text before the image.</p>
+        <p><img src="data:image/png;base64,#{encoded.strip}"></p>
```

Comment:
Just a minor detail: 
```suggestion
        <p><img src="data:image/jpeg;base64,#{encoded.strip}"></p>
```

Reply:
I tried both in Chrome and FF that when I do the following:
1. Go to quilljs.com
2. In another tab, go to google image search
3. Find any image (e.g. quilljs logo)
4. Copy the image using right mouse button
5. Paste it in the quill editor in the first tab

Here's what the outcome looks like in FF (it's the same in Chrome):
![Image type](https://user-images.githubusercontent.com/864340/193753207-50493016-3823-42c4-88d1-c40b0fbed3e3.png)

So I think the current examples are actually correct. I think when copy-pasting an image, the idea is to copy it with lossless compression.

Could you take another look?

Reply:
Google image is giving you a PNG image by default (`image.png`).

What I say is that using a JPEG file, like the one from the example (line 87), the MIME type should be the same. I just checked it in QuillJS too with `avatar.jpg`: 

![image](https://user-images.githubusercontent.com/717367/194050401-e983bfe5-77fb-42f4-bede-d6d33ee61c3c.png)

I should have explained it better, sorry about that